### PR TITLE
fix: post report messages messed up

### PR DESCRIPTION
**Pull request by nensidosari**
https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1726462490868179

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

**Review by rebelchris — APPROVED** [review]

**Comment by nensidosari** [comment]
we're ok to merge this without @capJavert 's approval?

**Review by capJavert — APPROVED** [review]
Yeah, feel free to merge this kind if one or two other team members approves.

**Pull request merged**
